Assets/Shopify/UIToolkit/Shops/Generic/Classes/CartItemView.cs

```diff
@@ -1,67 +1,71 @@
 namespace Shopify.UIToolkit {
     using Shopify.Unity;
     using Shopify.Unity.SDK;
+    using Shopify.UIToolkit.Shops.Generic;
     using System.Collections.Generic;
     using System.Linq;
     using System;
     using UnityEngine;
+    using UnityEngine.Events;
     using UnityEngine.UI;
 
-    /// <summary>
-    /// A View Model for converting the CartItem data into a format for the CartItemView component.
-    /// </summary>
-    public struct CartItemViewModel {
```

Comment:
This view model was a bit overkill for what we needed. It also got in the way of passing in the actual model to the line item view so we can know which product variant to increase/decrease quantity for.

Reply:
could always introduce a presenter if you need a layer to transform data in